app/assets/stylesheets/orders.css.scss

```diff
@@ -3,6 +3,8 @@
   margin-top: 10%;
   margin-left: auto;
   margin-right: auto;
+  border: solid 2px #363636;
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`